src/Core/Prefab/GlobeView.js

```diff
@@ -186,12 +248,21 @@ function GlobeView(viewerDiv, coordCarto) {
     this.wgs84TileLayer = wgs84TileLayer;
 
     this.mainLoop.addEventListener('command-queue-empty', () => {
-        viewerDiv.dispatchEvent(new CustomEvent('globe-built'));
+        this.dispatchEvent({ type: 'globe-built' });
     });
 
+    const fn = () => {
+        this.removeEventListener('globe-built', fn);
+        this.dispatchEvent({ type: 'initialized' });
+    };
+
+    this.addEventListener('globe-built', fn);
+
     window.addEventListener('resize', () => {
         this.controls.updateCamera(this.camera, this.viewerDiv.clientWidth, this.viewerDiv.clientHeight);
     }, false);
+
+    this.notifyChange(0, true);
```

Comment:
My original question wasn't precise enough.
Let me try again: "why is this added, since until now it worked fine without it?"
And after reading more of the changes, I can answer to my question: "Because until now, there was a call to apiGlobe.update() which actually does the this.notifyChange(..) for you"